spec/presenters/hyrax/iiif_manifest_presenter_spec.rb

```diff
@@ -217,6 +217,10 @@
       expect(presenter.version).to be_a String
     end
 
+    it 'changes when the work changes' do
+      expect { work.save }.to change { presenter.version }
```

Comment:
i originally wrote a test like this, but found it failed because the object saved within a second of when it was originally created. since i expect this method to continue to change, i decided not to belabor this issue.

it looks like maybe this version is failing for the same reason?